code/datums/ftl_ships/ship_weapons.dm

```diff
@@ -146,6 +146,33 @@
 	playsound(epicenter, 'sound/magic/lightningbolt.ogg', 100, 1)
 	empulse(epicenter,5,10,1)
 
+
+/datum/ship_attack/carrier_weapon
+	cname = "Carrier Blaster"
+	projectile_effect = "leaper"
+	hull_damage = 0
+	shield_bust = 1
+	var/list/boarding_mobs = list(/mob/living/simple_animal/hostile/droid)
+	var/amount = 5
+
+/datum/ship_attack/carrier_weapon/damage_effects(turf/epicenter)
+
+	playsound(epicenter, 'sound/ftl/shipweapons/carrier_hit.ogg', 100, 1)
+	for(var/I = 1 to amount)
+		var/path = pick(boarding_mobs)
+		var/mob/to_spawn = new path(epicenter)
+		to_spawn.faction = list(our_ship_component.ship.faction)
```

Comment:
Yes it is, thanks. the reason this is happening by the way is because I changed all components to ship_component and this had conflicts and i lazily resolved them :^)